clippy_lints/src/doc/empty_docs.rs

```diff
@@ -0,0 +1,25 @@
+use clippy_utils::diagnostics::span_lint_and_help;
+use rustc_ast::Attribute;
+use rustc_lint::LateContext;
+
+use super::EMPTY_DOCS;
+
+// TODO: Adjust the parameters as necessary
+pub(super) fn check(cx: &LateContext<'_>, attrs: &[Attribute]) {
+    let doc_attrs: Vec<_> = attrs.iter().filter(|attr| attr.doc_str().is_some()).collect();
+
+    let span;
+    if let Some(first) = doc_attrs.first()
+        && let Some(last) = doc_attrs.last()
+    {
+        span = first.span.with_hi(last.span.hi());
```

Comment:
If you pass the fragments in here rather than the raw attributes you can use https://doc.rust-lang.org/nightly/nightly-rustc/rustc_resolve/rustdoc/fn.span_of_fragments.html to get the span